### PR TITLE
feat(settings): teamProjects and projectTeams dropdown styles

### PR DESCRIPTION
**Pull request by Chrissy**
temprojecteams

### CODE REVIEW

File: src/sentry/static/sentry/app/components/userListElement.jsx [hunk 1]
```diff
@@ -1,40 +0,0 @@
-import PropTypes from 'prop-types';
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/userListElement.jsx]: this is only used in one place for the foreseeable future, so i just moved it there (`teamMembers.jsx`)

File: src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx [hunk 2]
```diff
@@ -162,6 +159,42 @@ class ProjectTeams extends AsyncView {
...
+        value: team.id,
+        searchKey: team.slug,
+        label: <TeamDropdownElement>#{team.slug}</TeamDropdownElement>,
+      }));
+
+    return (
+      <DropdownAutoComplete
+        items={teamsToAdd}
+        onSelect={this.handleAdd}
+        action={
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx]: need to move this over to `menuHeader`

File: src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx [hunk 3]
```diff
@@ -162,6 +159,42 @@ class ProjectTeams extends AsyncView {
     });
   };
 
+  renderAddTeamButton() {
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx]: most of this is just taken from the primary render call. the only thing i added is the jsx label and (soon) a better action button.

File: src/sentry/static/sentry/app/icons/icon-trash.svg [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/icons/icon-trash.svg]: i'm not actually using this, but I figure we'll need a trash icon one of these days.

**Review by MaxBittker — APPROVED** [review]
tried it locally, looks very nice and works well 🍰

File: src/sentry/static/sentry/app/components/userListElement.jsx (hunk 1 above)
Comment (MaxBittker) [reply src/sentry/static/sentry/app/components/userListElement.jsx]: 👍

File: src/sentry/static/sentry/app/components/dropdownButton.jsx [hunk 5]
```diff
@@ -7,8 +7,8 @@ import InlineSvg from './inlineSvg';
 const DropdownButton = ({isOpen, children, ...props}) => {
   return (
     <StyledButton isOpen={isOpen} {...props}>
-      <StyledChevronDown />
       {children}
+      <StyledChevronDown />
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/dropdownButton.jsx]: 👍

File: src/sentry/static/sentry/app/icons/icon-trash.svg (hunk 4 above)
Comment (MaxBittker) [reply src/sentry/static/sentry/app/icons/icon-trash.svg]: 🗑 cool, doesn't hurt

**Comment by Chrissy** [comment]
@MaxBittker thanks!!

**Pull request merged**